healthcheck

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 HOST PORT
+
+Healthcheck for the RT docker container."
```

Comment:
I think multiple `echo`s would be better than bad indentation.

Reply:
I'd agree on this

Reply:
Done